src/signals/incident-management/containers/IncidentOverviewPage/saga.js

```diff
@@ -1,15 +1,16 @@
 import { all, put, select, takeLatest } from 'redux-saga/effects';
 import { push } from 'react-router-redux';
 
-import { authCall } from 'shared/services/api/api';
+import { authCall, authPostCall, authDeleteCall } from 'shared/services/api/api';
 import CONFIGURATION from 'shared/services/configuration/configuration';
 
-import { REQUEST_INCIDENTS, INCIDENT_SELECTED } from './constants';
-import { requestIncidentsSuccess, requestIncidentsError, filterIncidentsChanged, pageIncidentsChanged, sortIncidentsChanged } from './actions';
+import { REQUEST_INCIDENTS, INCIDENT_SELECTED, GET_FILTERS, REMOVE_FILTER, REVERT_FILTER } from './constants';
+import { requestIncidentsSuccess, requestIncidentsError, filterIncidentsChanged, pageIncidentsChanged, sortIncidentsChanged,
+  getFiltersSuccess, getFiltersFailed, removeFilterSuccess, removeFilterFailed, revertFilterSuccess, revertFilterFailed } from './actions';
 import { makeSelectFilterParams } from './selectors';
 
 export function* fetchIncidents(action) {
-  const requestURL = `${CONFIGURATION.API_ROOT}signals/v1/private/signals`;
+  const requestURL = `${CONFIGURATION.API_ROOT}signals/v1/private/signals/`;
```

Comment:
How do the filters actually work? Filters seem to be set in the back-end. But I'm not seing any reference to the current filter set here.

Reply:
the filter is created in the FilterForm that is wrapped in the Filter container. 
the active filter state is in de redux state of IncidentOverviewPage.filter.